ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift

```diff
@@ -13,6 +13,9 @@ class ButtonDemoController: DemoController {
 
         container.alignment = .leading
 
+        addTitle(text: "SwiftUI Demo")
```

Comment:
minor: Should this be `SwiftUI Button Demo` since it's showing the `ButtonDemoControllerSwiftUI`?

Reply:
I'm comfortable either way--this is within the `ButtonDemoController` already, so the context should be clear, but I can add it for additional clarity.